### PR TITLE
feat(twap-orders): integrate TradeWidget in Swap, Limit orders and Advanced orders

### PR DESCRIPTION
**Pull request by shoom3301**
This PR is a next step from https://github.com/cowprotocol/cowswap/pull/2400

# Summary

1. Integrated `TradeWidget` into Swap widget
2. Integrated `TradeWidget` into Limit orders widget
3. Integrated `TradeWidget` into Advanced orders widget
4. Updated selectors in e2e tests

  # To Test

For swap and Limit orders

1. Wrap/unwrap
2. Set/remove recipient
3. Import token
4. Eth-flow
5. Siwtch tokens places

### CODE REVIEW

File: src/cow-react/common/pure/CurrencyInputPanel/types.ts [hunk 1]
```diff
@@ -3,12 +3,12 @@ import { Currency, CurrencyAmount } from '@uniswap/sdk-core'
 import { ReceiveAmountInfo } from '@cow/modules/swap/helpers/tradeReceiveAmount'
 
 export interface CurrencyInfo {
-  field: Field
   label?: string
-  viewAmount: string
+  field: Field
+  currency: Currency | null
   rawAmount: CurrencyAmount<Currency> | null
+  viewAmount: string // TODO: should be calculated in CurrencyInputPanel
```
Comment (anxolin) [inline src/cow-react/common/pure/CurrencyInputPanel/types.ts:10]: I don't get the TODO. Also, you just re-ordered the fields, right? i didn't notice any change

File: src/cow-react/modules/trade/containers/TradeWidget/index.tsx [hunk 2]
```diff
@@ -110,54 +76,70 @@ export function TradeWidget(props: TradeWidgetProps) {
...
-        />
-        {showRecipient && (
-          <styledEl.StyledRemoveRecipient recipient={recipient || ''} onChangeRecipient={onChangeRecipient} />
+        {lockScreen ? (
+          lockScreen
+        ) : (
+          <>
+            <div>
+              <CurrencyInputPanel
+                id="input-currency-input"
```
Comment (anxolin) [inline src/cow-react/modules/trade/containers/TradeWidget/index.tsx:92]: Would it be worth it to do it as a const, given is a param, and heavely used and repeated in the e2e?

File: src/cow-react/common/pure/CurrencyInputPanel/types.ts (hunk 1 above)
Comment (shoom3301) [reply src/cow-react/common/pure/CurrencyInputPanel/types.ts:10]: https://github.com/cowprotocol/cowswap/pull/2404
Here is a fix

File: src/cow-react/modules/trade/containers/TradeWidget/index.tsx (hunk 2 above)
Comment (shoom3301) [reply src/cow-react/modules/trade/containers/TradeWidget/index.tsx:92]: It's a good idea! Will add it in the next PR, don't you mind?

**Comment by elena-zh** [comment]
Hey @shoom3301 , great job! 
Same issues as in https://github.com/cowprotocol/cowswap/pull/2404#pullrequestreview-1409076718

**Review by alfetopito — APPROVED** [review]

**Comment by alfetopito** [comment]
Actually, integration tests are failing because some tests are still looking for `limit` and `swap` ids

File: src/cow-react/modules/trade/containers/TradeWidget/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/cow-react/modules/trade/containers/TradeWidget/index.tsx:92]: is not too important. Fine if we don't do it actually. Soft suggestion

**Review by anxolin — APPROVED** [review]

**Pull request merged**